lib/label_printer/label_printer/label/batch_tube.rb

```diff
@@ -32,7 +32,7 @@ def round_label_top_line(tube)
       end
 
       def round_label_bottom_line(tube)
-        tube.is_a?(PacBioLibraryTube) ? source_plate_barcode(tube).split('').last(4).join : super
+        tube.is_a?(PacBioLibraryTube) ? source_plate_barcode(tube)[-4..] : super
       end
```

Comment:
I made a similar fix when updating rails in another branch, this actually doesn't quite work:
```
irb(main):022:0> 'abcdefghijk'[-4..]
=> "hijk"
irb(main):023:0> 'ijk'[-4..]
=> nil
```
I don't know what the risk of us getting strings of three characters here, but I decided to play it safe and went with:
```
      def round_label_bottom_line(tube)
        if tube.is_a?(PacBioLibraryTube)
          source_plate = source_plate_barcode(tube)
          # Return the last 4 characters. The fallback handles scenarios where
          # we have fewer than 4 characters in the string, as:
          #     'abc'[-4..] # => nil
          source_plate[-4..] || source_plate
        else
          super
        end
      end
      ```